<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1126</width>
    <height>635</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
	font-family: Candara, Calibri, loma; 
	font-size:14pt;
 	font-weight:400; 
	font-style:normal;
	background: rgba(32, 80, 96, 100);
	 /*background-color: #404E71;#21315E;#454545;*/
	border: 1px solid black;
}

QPushButton {
	background-color: red;
	border-style: outset;
	border-width: 2px;
	border-radius: 10px;
	border-color: beige;
 	font: bold 14px;
	min-width: 10em;
	padding-top: 6px;
	padding-bottom: 6px;
	padding-left: 2px;
	padding-right: 2px;
}

QLineEdit, QSpinBox, #console_pte {
    color: rgb(39, 101, 206);
    background-color: rgb(255, 255, 255);
    selection-color: white;
    selection-background-color: rgb(191, 31, 127);
    border: 2px groove gray;
    /*border-radius: 10px;*/
    padding: 2px 4px;
}

QLabel{ 
	/*color: #8f8f8f; */
	background: transparent;
	font-weight: bold;
	color: white;
	border: none; 
	margin: 6px; 
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLineEdit" name="input_text_led">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>38</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>40</height>
       </size>
      </property>
      <property name="text">
       <string>Writing at 0x00030000... (52 %)</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLineEdit" name="search_regex_led">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>38</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>40</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QPushButton" name="search_button">
        <property name="minimumSize">
         <size>
          <width>228</width>
          <height>38</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>128</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string>SEARCH</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLineEdit" name="extract_regex_led">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>38</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string>(\d+)\s*%</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2" rowspan="2">
       <widget class="QPushButton" name="extract_button">
        <property name="minimumSize">
         <size>
          <width>228</width>
          <height>38</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>128</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string>EXTRACT</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" rowspan="2" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLineEdit" name="replace_regex_led">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>38</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>40</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>38</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>40</height>
           </size>
          </property>
          <property name="text">
           <string>Replace by:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="replace_by_text_led">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>38</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>40</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="2">
       <widget class="QPushButton" name="replace_button">
        <property name="minimumSize">
         <size>
          <width>228</width>
          <height>38</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>128</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string>REPLACE</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="console_pte"/>
    </item>
    <item>
     <widget class="QFrame" name="bottom_frame">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>64</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>96</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame{
	border: 0;
	background: transparent;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>619</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="quit_button">
         <property name="minimumSize">
          <size>
           <width>228</width>
           <height>38</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>128</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string>QUIT</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="apply_button">
         <property name="minimumSize">
          <size>
           <width>228</width>
           <height>38</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>128</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string>APPLY</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1126</width>
     <height>37</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
